Code/Tools/ProjectManager/Platform/Mac/ProjectUtils_mac.cpp

```diff
@@ -7,15 +7,59 @@
 
 #include <ProjectUtils.h>
 
+#include <QProcess>
+
 namespace O3DE::ProjectManager
 {
     namespace ProjectUtils
     {
-        AZ::Outcome<void, QString> FindSupportedCompilerForPlatform()
+        AZ::Outcome<QProcessEnvironment, QString> GetCommandLineProcessEnvironment()
         {
-            // Compiler detection not supported on platform
-            return AZ::Success();
+            // For CMake on Mac, if its installed through home-brew, then it will be installed
+            // under /usr/local/bin, which may not be in the system PATH environment. 
+            // Add that path for the command line process so that it will be able to locate
+            // a home-brew installed version of CMake
+            QProcessEnvironment currentEnvironment(QProcessEnvironment::systemEnvironment());
+            QString pathValue = currentEnvironment.value("PATH");
+            pathValue += ":/usr/local/bin";
+            currentEnvironment.insert("PATH", pathValue);
+            return AZ::Success(currentEnvironment);
         }
-        
+
+        AZ::Outcome<QString, QString> FindSupportedCompilerForPlatform()
+        {
+            QProcessEnvironment currentEnvironment(QProcessEnvironment::systemEnvironment());
+            QString pathValue = currentEnvironment.value("PATH");
+            pathValue += ":/usr/local/bin";
+            currentEnvironment.insert("PATH", pathValue);
```

Comment:
I'm not sure PATH(or other environment variables) is set when launching an app from Finder. This thread from Apple forums isn't about the same env vars but it's essentially talking about the same thing I think(setting environment variables for GUI i.e., finder launched apps): https://developer.apple.com/forums/thread/74371?answerId=217368022#217368022
I haven't found anything from Apple's official docs about this(other than the forum posts) but I've found that environment variables haven't worked when launching from Finder in general(at least not when the app is downloaded and system integrity protection and other security stuff is on).

Reply:
Never mind. I just looked at ExecuteCommandResult() and it sets the environment to which you added `/usr/local/bin` as the environment before launching any of the commands. So this is fine.

Reply:
Yes. The calls to execute "which cmake" will fail(unless it is also found in /usr/local/bin)when launched from finder. But it should still be able to execute cmake as long as it's installed in /usr/local/bin since we added that to PATH and then supplied it as part of the environment when launching the commands.

Reply:
Actually, 'which' works on the execute, the QProcessEnvironment is initialized with a copy of the currentSystem environment, which has the PATH set up. But while debugging, I didnt see /usr/local/bin in the path (the other paths were there), so I added it here